adapters/suntContent/suntContent_test.go

```diff
@@ -0,0 +1,215 @@
+package suntContent
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/mxmCherry/openrtb/v16/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderSuntContent, config.Adapter{
+		Endpoint: "https://mockup.suntcontent.com/",
+	}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "suntContenttest", bidder)
+}
+
+func TestResolvePriceMacro(t *testing.T) {
+	_, buildErr := Builder(openrtb_ext.BidderSuntContent, config.Adapter{
```

Comment:
Why are you invoking the builder in all the tests when you don't seem to be using it?

Reply:
Haha, no idea actually - probably just copied the example from the doc and underscored the unused variables at some point.

I removed the redundant parts in the Seeding Alliance and SUNT adapter.